cairo-sys-rs/src/lib.rs

```diff
@@ -58,6 +58,7 @@ pub type CairoSvgUnit = c_int;
 pub type CairoTextClusterFlags = c_int;
 
 pub type cairo_antialias_t = CairoAntialias;
+pub type cairo_font_type_t = CairoFontType;
```

Comment:
Maybe in -sys we should only have the names as in C, and the CamelCase enum types only in the non-sys crate? That's more consistent with everything else.

Reply:
Agree.
@GuillaumeGomez What you think?

Reply:
I agree as well.